Frontify/arcade-tokens
# feat: tailwind upgrade v3

### How to test
1. do a build on branch `next` 
2. do a build on branch `feat/tailwind-upgrade-v3`

Compare build's, only expected differences are:
- timestamps in file's headers
- tailwind.config (dropped non-necessary params)
